controllers/changetierrequest/changetierrequest_controller.go

```diff
@@ -176,6 +177,7 @@ func (r *Reconciler) changeTier(logger logr.Logger, changeTierRequest *toolchain
 			// here we remove the template hash label since it was change for one or all target clusters
 			delete(mur.Labels, nstemplatetier.TemplateTierHashLabelKey(mur.Spec.UserAccounts[i].Spec.NSTemplateSet.TierName))
 			mur.Spec.UserAccounts[i].Spec.NSTemplateSet = newNsTemplateSet
+			mur.Spec.TierName = changeTierRequest.Spec.TierName
```

Comment:
it's the same. 
Actually, both of them should be replaced by one assignment outside of the for-loop.

Reply:
done in 9847f5d